packages/circuits/circuits/dataProof.circom

```diff
@@ -1,9 +1,7 @@
-pragma circom 2.0.0;
+pragma circom 2.1.0;
 
-include "../../../node_modules/@unirep/circuits/circuits/proveReputation.circom";
-include "../../../node_modules/@unirep/circuits/circuits/circomlib/circuits/poseidon.circom";
-include "../../../node_modules/@unirep/circuits/circuits/circomlib/circuits/mux1.circom";
-include "../../../node_modules/@unirep/circuits/circuits/circomlib/circuits/gates.circom";
+include "../../../node_modules/@unirep/circuits/circuits/hasher.circom";
```

Comment:
And I believe "../../../node_modules/" could be removed to make the style cleaner.